lib/core/utils.ts

```diff
@@ -22,6 +22,8 @@ import * as cdk from 'aws-cdk-lib';
 import * as iam from 'aws-cdk-lib/aws-iam';
 
 import { Config } from '../schema';
+import { Effect, ManagedPolicy, PolicyDocument, PolicyStatement, Role, ServicePrincipal } from 'aws-cdk-lib/aws-iam';
```

Comment:
nitpick on duplicate iam imports- likely IDE added both- i'd try removing the star import instead of this one